community/captains-log/index.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: single
+title:  "Captain's Log"
+sidebar:
+  nav: community
+github_events:
+  image_path: assets/images/graphs/GithubEvents_graph.png
+  alt: "Graph of Github Events Since Open-Sourced"
+activity:
+  image_path: assets/images/graphs/ActivityByRepo_graph.png
+  alt: "Graph of Github Activity by Repository Since Open-Sourced"
+pr_authors:
+  image_path: assets/images/graphs/PRAuthorsUnique_graph.png
+  alt: "Graph of Unique PR Authors Since Open-Sourced"
+---
+
+You may have heard [success stories](/success-stories/) of enterprises moving from brittle deployments to continuous delivery with Spinnaker:
+* [Airbnb uses Spinnaker](https://techbeacon.com/app-dev-testing/how-airbnb-scaled-its-migration-continuous-delivery-spinnaker) to migrate from monolith to service-oriented architecture.
+* [SAP leverages Spinnaker](https://blog.spinnaker.io/pipeline-redemption-how-spinnaker-is-shaping-delivery-excellence-at-sap-3b3c931b4f63?) on its mission to run the world better.
+* [Pinterest uses Spinnaker](https://devops.com/devops-chat-ci-cd-velocity-for-large-monolithic-services-with-pinterest/) to boost productivity as it pioneers visual discovery.
+* [Mercari champions Spinnaker](https://speakerdeck.com/tcnksm/continuous-delivery-for-microservices-with-spinnaker-at-mercari) as a safeguard against deployment fear while releasing new services.
+* [Salesforce adopted Spinnaker](https://engineering.salesforce.com/salesforce-speakers-at-spinnaker-summit-and-kubecon-2019-d968292fd681) to bake images for both Kubernetes and VMs, to support its complex delivery requirements.
+
+### But what are the numbers? How big is the project? What’s the growth trajectory?
+
+## Spinnaker's Trajectory
+
+{%
+  include
+  figure
+  image_path="./stats-2020-02-05-with-logo.png"
+%}
+
+The Spinnaker community began as a partnership between Netflix and Google, eager to share the benefits of Continuous Delivery. It has grown into a vibrant Open Source project, attracting hundreds of companies to participate as it evolves, integrating unique use cases and tool chains. Check it out!
```

Comment:
Yeah I went back and forth on this. The placement didn't look great to me with too much text at the top, I didn't want 2 paragraphs before getting to the meat. I can play with it more tho

Reply:
K I shortened the copy so it's just 2 lines and keeps the graphic "above the fold" to catch the eye. Thoughts on "Behold" instead of "check it out"? Is lol